collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -1199,7 +1298,7 @@ def create_lines(database, lines):
         'lines': create_lines(database_name, chart_template['lines'])}
 
 
-def add_replication_delta_chart(order, definitions, name, application_name):
+def add_replication_standby_chart(order, definitions, name, application_name, chart_description):
```

Comment:
> chart_description

That is [chart family](https://learn.netdata.cloud/docs/agent/collectors/plugins.d#chart)

Reply:
thanks